docs/tutorials/testing/testing_locally.md

```diff
@@ -45,25 +45,23 @@ Though functional, we find this approach to be somewhat slow for the purposes of
 :::
 
 ## ParrelSync
+[**ParrelSync**](https://github.com/VeriorPies/ParrelSync) is an open-source Unity editor extension that allows users to **test multiplayer gameplay without building the project** by having another Unity editor window opened and mirror the changes from the original project. 
+
 :::caution
 
-ParallelSync is **not** supported by Unity.  More information on its usage is available [here](https://github.com/VeriorPies/ParrelSync). Troubleshooting information can be found [here](https://github.com/VeriorPies/ParrelSync/wiki/Troubleshooting-&-FAQs)
+ParallelSync is **not** supported by Unity.  Please see the repositor for more information and [troubleshooting information](https://github.com/VeriorPies/ParrelSync/wiki/Troubleshooting-&-FAQs)
 
 :::
 
-![parrelsync-bossroom-demo](../../../static/img/parrelsync-bossroom-demo.gif)
-
-[**ParrelSync**](https://github.com/VeriorPies/ParrelSync) is an open-source Unity editor extension that allows users to **test multiplayer gameplay without building the project** by having another Unity editor window opened and mirror the changes from the original project. 
-
 **ParrelSync** works by making a copy of the original project folder and creating symbolic links to the `Asset` and `Project Settings` folders back from the original project.
 
 We use **ParrelSync** for local iteration in [Boss Room sample](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/).
 
-:::important
+![parrelsync-bossroom-demo](../../../static/img/parrelsync-bossroom-demo.gif)
 
-**ParrelSync** relies on symbolic links and partial copies of the original project folder structure - generally it is completely safe. 
+:::important
 
-To ensure that no bug in any of the software you use can destroy your work - it is recommended that you consistently backup your project or use a version control system. Some common examples are:
+**ParrelSync** relies on symbolic links and partial copies of the original project folder structure.  It is generally completely safe, but it is recommended that you consistently backup your project or use a version control system. Some common examples are:
```

Comment:
this does not read correctly.

Reply:
@Briancoughlin - Do you have a suggestion for a change?  I'm not sure which part you think is unclear.  The second sentence?  

I'm just trying to fix it up based on the previous version which I think was even less clear.

Reply:
I'm not sure which part @Briancoughlin is referring to specifically, but I might reword the last two paragraphs like so:

> **ParrelSync** is generally safe because it relies on symbolic links and partial copies of the original project folder structure.

> :::important
> Although ParrelSync is generally safe, you should still consistently back up your project or use a version control system to ensure you don't lose your work.